### PR TITLE
[RFR] Preparations for new graphics

### PR DESCRIPTION
**Pull request by touv**

### CODE REVIEW

**Review by Kmaschta — APPROVED** [review]

File: src/common/transformers/GET.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import _get from 'lodash.get';
+import { transformerWithArg } from './transformer';
+
+export const get = (value, path) => {
+    if (!value) {
```
Comment (ThieryMichel) [inline src/common/transformers/GET.js]: I think it's not needed as _.get accept a default return value as the third parameter, and it should cover the case when we have no value.

File: src/common/transformers/GET.spec.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import expect from 'expect';
+
+import { get } from './GET';
+
+describe('GET', () => {
+    it("should return key's value from object", () => {
+        expect(get({ a: { aa: 1 }, b: 2 }, 'a.aa')).toEqual(1);
+    });
```
Comment (ThieryMichel) [inline src/common/transformers/GET.spec.js:8]: test with no value, incorrect path, and no path too.

**Review by ThieryMichel — CHANGES_REQUESTED** [review]

File: src/common/transformers/GET.js (hunk 1 above)
Comment (touv) [reply src/common/transformers/GET.js]: ok, this test is useless, i will drop it

File: src/common/transformers/GET.spec.js (hunk 2 above)
Comment (touv) [reply src/common/transformers/GET.spec.js:8]: I do not think it is useful to thoroughly test the lodash function, the value is elsewhere

**Review by Kmaschta — APPROVED** [review]

**Pull request merged**